packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -206,21 +218,21 @@ contract MerkleDistributor is Ownable {
     }
 
     // Verify claim is valid and mark it as completed in this contract.
-    function _verifyAndMarkClaimed(Claim memory claim) private {
+    function _verifyAndMarkClaimed(Claim memory _claim) private {
         // Check claimed proof against merkle window at given index.
-        require(verifyClaim(claim), "Incorrect merkle proof");
+        require(verifyClaim(_claim), "Incorrect merkle proof");
         // Check the account has not yet claimed for this window.
-        require(!isClaimed(claim.windowIndex, claim.accountIndex), "Account has already claimed for this window");
+        require(!isClaimed(_claim.windowIndex, _claim.accountIndex), "Account has already claimed for this window");
 
         // Proof is correct and claim has not occurred yet, mark claimed complete.
-        _setClaimed(claim.windowIndex, claim.accountIndex);
+        _setClaimed(_claim.windowIndex, _claim.accountIndex);
         emit Claimed(
             msg.sender,
-            claim.windowIndex,
-            claim.account,
-            claim.accountIndex,
-            claim.amount,
-            address(merkleWindows[claim.windowIndex].rewardToken)
+            _claim.windowIndex,
```

Comment:
ALl of these `claim` -> `_claim` renamings are to eliminate compiler warnings about variable shadowing